rest-api/src/main/java/gov/cms/qpp/conversion/api/services/VersionServiceImpl.java

```diff
@@ -0,0 +1,15 @@
+package gov.cms.qpp.conversion.api.services;
+
+import org.springframework.stereotype.Service;
+
+import com.jcabi.manifests.Manifests;
+
+@Service
+public class VersionServiceImpl implements VersionService {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public class by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)